src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -243,9 +243,9 @@ public override bool IsEnabled(string name)
         /// <summary>
         /// Override abstract method
         /// </summary>
-        public override bool IsEnabled(string name, object arg1, object arg2 = null)
+        public override bool IsEnabled(string name, object? arg1, object? arg2 = null)
```

Comment:
can the name be `null`?

Reply:
Nothing found to tell it is nullable or not null, but from context it doesn't make sense to be null and sample app throwing when null passed